actions/dependency/update-from-metadata/action.yml

```diff
@@ -0,0 +1,59 @@
+name: 'Update buildpack.toml with new dependencies from metadata file'
+
+description: |
+  This action updates the buildpack.toml from a metadata JSON file.
+
+inputs:
+  buildpack_toml_path:
+    description: 'absolute path to buildpack.toml'
+    default: 'buildpack.toml'
+  metadata_file_path:
+    description: 'absolute path to metadata.json'
+    default: 'metadata.json'
```

Comment:
Do these inputs need to be absolute paths if the defaults are relative?

Reply:
You're right. It can be either relative or absolute.